Documentation/RVEngine/Host/LOADER.md

```
@@ -0,0 +1,143 @@
+public:
+    void* start_address;
+    size_t size;
+    
+    bool contains(void* address);
+    // Сканирование секции
+};
+```
+
+## Процесс сканирования
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)